htdocs/core/class/html.formticket.class.php

```diff
@@ -50,9 +50,15 @@ class FormTicket
 
 	/**
 	 * @var string	The track_id of the ticket. Used also for the $keytoavoidconflict to name session vars to upload files.
+	 * @deprecated
 	 */
 	public $track_id;
 
+	/**
+	 * @var string trackid
+	 */
+	public $trackid;
```

Comment:
track_id and trackid ae 2 different vars.
track_id is a column of ticket containing a second id in format of a hash of ticket. 
trackid is the key used for the tracking of emails sents